Src/Web/Web.Shared.Net/ApplicationInsightsHttpModule.cs

```diff
@@ -67,7 +88,16 @@ private void OnBeginRequest(object sender, EventArgs eventArgs)
         {
             if (this.isEnabled)
             {
-                this.TraceCallback("OnBegin", (HttpApplication)sender);
+                HttpApplication httpApplication = (HttpApplication)sender;
```

Comment:
Just curious if this cast is always true or if using "as HttpApplication" would be safer? (If it's always true then leave it this way)